### PR TITLE
cleans up syntax and style for attachments.md

### PR DESCRIPTION
**Pull request by ablwr**
Attachments.md is pretty sparse. I removed the header for Font Files since there was nothing in it, but maybe I should ask the list for opinions and potential spec guidelines?

### CODE REVIEW

**Comment by dericed** [comment]
The `Font Files` header goes with https://github.com/Matroska-Org/matroska-specification/pull/115, which I haven't gotten around to completing.

**Comment by ablwr** [comment]
Ahhhh, whoops. Well, if it gets removed in this PR, it should come back in yours, which makes sense semantically. But if you want me to add it back, LMK.

File: attachments.md [hunk 1]
```diff
@@ -5,21 +5,21 @@
 
 ## Introduction
 
-Matroska supports storage of related files and data in the Attachments Top-Level Element. Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancilliary files related to the Segment.
+Matroska supports storage of related files and data in the `Attachments Element` (a `Top-Level Element`). Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancillary files related to the `Segment`.
```
Comment (mbunkus) [inline attachments.md]: > Attachments can be used…

Could probably be "`Attachments` can be used…" as we're talking about the actual elements. So far we haven't defined what the feature "attachment" is, only what the `Attachment Element` can do. Yes, it's a philosophical distinction. So sue me! 😁

File: attachments.md [hunk 2]
```diff
@@ -5,21 +5,21 @@
...
 
 `Matroska Readers` MUST NOT execute files stored as Attachments.
 
 ## Cover Art
 
-This section defines a set of guidelines for the storage of cover art in Matroska files. A `Matroska Reader` MAY use embedded cover art to display a representation still-image depiction of the multimedia contents of the Matroska file.
+This section defines a set of guidelines for the storage of cover art in Matroska files. A `Matroska Reader` MAY use embedded cover art to display a representational still-image depiction of the multimedia contents of the Matroska file.
 
-Cover art SHOULD only use the JPEG and PNG picture formats.
+Cover art SHOULD only use JPEG and PNG formats.
```
Comment (mbunkus) [inline attachments.md]: Hmm, this sounds a bit… strange. Covert art (images) can not use a format (in the sense that "use" is an active verb). An encoder can use a specific format. Or a picture can be encoded in a format. Maybe something like "Only JPEG and PNG image formats SHOULD be used for cover art pictures." OK, doesn't sound that much better; hopefully I can still get my point across.

File: attachments.md [hunk 3]
```diff
@@ -5,21 +5,21 @@
...
 
-There can be 2 different covers for a movie/album. A portrait one (like a DVD case) and a landscape one (like a banner ad for example, looking better on a wide screen).
+There can be 2 different covers for a movie/album. A portrait style (e.g., a DVD case) and a landscape style (e.g., a wide banner ad).
```
Comment (mbunkus) [inline attachments.md]: "two" instead of "2"? In German we usually use words for numbers up to and including 12. Is there something similar in English?

Comment (mbunkus) [inline attachments.md]: The second sentence is not a complete sentence. It a verb ;)

Proposal: "…a movie/album: a portrait style…"

File: attachments.md [hunk 4]
```diff
@@ -5,21 +5,21 @@
...
 
-There can be 2 versions of the same cover, the `normal cover` and the `small cover`. The dimension of the `normal cover` SHOULD be 600 on the smallest side (for example, 960x600 for landscape, 600x800 for portrait, or 600x600 for square). The dimension of the `small cover` SHOULD be 120 on the smallest side (for example, 192x120 or 120x160).
+There can be 2 versions of the same cover, the `normal cover` and the `small cover`. The dimension of the `normal cover` SHOULD be 600 pixels on the smallest side (for example, 960x600 for landscape, 600x800 for portrait, or 600x600 for square). The dimension of the `small cover` SHOULD be 120 pixels on the smallest side (for example, 192x120 or 120x160).
```
Comment (mbunkus) [inline attachments.md]: "two" vs "2" again

File: attachments.md [hunk 5]
```diff
@@ -5,21 +5,21 @@
...
 
-Versions of cover art can be differentiated by the filename, which is stored in the `FileName Element`. The default filename of the `normal cover` in square or portrait mode is `cover.(jpg|png)`. When stored, the `normal cover` SHOULD be the first Attachment in storage order. The `small cover` SHOULD be prefixed with "small_", such as `small_cover.(jpg|png)`. The landscape variant SHOULD be suffixed with "_land", such as `cover_land.(jpg|png)`. The filenames are case sensitive and SHOULD all be lower case.
+Versions of cover art can be differentiated by the filename, which is stored in the `FileName Element`. The default filename of the `normal cover` in square or portrait mode is `cover.(jpg|png)`. When stored, the `normal cover` SHOULD be the first Attachment in storage order. The `small cover` SHOULD be prefixed with "small_", such as `small_cover.(jpg|png)`. The landscape variant SHOULD be suffixed with "\_land", such as `cover_land.(jpg|png)`. The filenames are case sensitive and SHOULD all be lowercase.
```
Comment (mbunkus) [inline attachments.md]: > …the first Attachment in storage…

→ `Attachment`

Comment (mbunkus) [inline attachments.md]: > …sensitive and SHOULD all be lowercase.

"…sensitive and SHOULD be all lowercase." ?? Really not sure about this.

File: attachments.md (hunk 4 above)
Comment (retokromer) [inline attachments.md]: (It’s the same in French, Italian and Romansh.)

File: attachments.md (hunk 1 above)
Comment (ablwr) [inline attachments.md]: :)

File: attachments.md (hunk 5 above)
Comment (ablwr) [inline attachments.md]: SHOULD we lift the recommendation? I'll do that.

**Comment by ablwr** [comment]
Made changes and squashed commits for easy re-review.

File: attachments.md [hunk 6]
```diff
@@ -5,21 +5,21 @@
...
 
-Versions of cover art can be differentiated by the filename, which is stored in the `FileName Element`. The default filename of the `normal cover` in square or portrait mode is `cover.(jpg|png)`. When stored, the `normal cover` SHOULD be the first Attachment in storage order. The `small cover` SHOULD be prefixed with "small_", such as `small_cover.(jpg|png)`. The landscape variant SHOULD be suffixed with "_land", such as `cover_land.(jpg|png)`. The filenames are case sensitive and SHOULD all be lower case.
+Versions of cover art can be differentiated by the filename, which is stored in the `FileName Element`. The default filename of the `normal cover` in square or portrait mode is `cover.(jpg|png)`. When stored, the `normal cover` SHOULD be the first Attachment in storage order. The `small cover` SHOULD be prefixed with "small_", such as `small_cover.(jpg|png)`. The landscape variant SHOULD be suffixed with "\_land", such as `cover_land.(jpg|png)`. The filenames are case sensitive.
```
Comment (robUx4) [inline attachments.md]: Is the extra `\` in `_land` intentional ?

Comment (retokromer) [inline attachments.md]: My guess: markdown… start of _italic_.

Comment (ablwr) [inline attachments.md]: The `\` escapes the `_` so it prints plainly instead of trying to start a _italics pattern_!

Comment (robUx4) [inline attachments.md]: The TXT and HTML (and XML) output seem correct so LGTM.

**Comment by ablwr** [comment]
👍 Unsure what constitutes consensus, merge at your leisure!

**Comment by mbunkus** [comment]
I understood both Steve and me to be fine with the changes. Therefore…

**Comment by ablwr** [comment]
Wooooo!! Now onto the much more robust diagram.md.

**Pull request merged**